python/rikai/spark/sql/codegen/sklearn.py

```diff
@@ -38,8 +43,8 @@ def sklearn_inference_udf(
     ) -> Iterator[pd.Series]:
         model = spec.load_model()
         for series in list(iter):
-            X = np.vstack(series.to_numpy())
-            y = model.predict(X)
+            X = np.vstack(series.apply(_pickler.loads).to_numpy())
+            y = [_pickler.dumps(pred.tolist()) for pred in model.predict(X)]
```

Comment:
We could consider turning generate_udf into a class so that we can take care of the pickling and the pandas_udf constructor call in the base class? This way the hack is more isolated

Reply:
sgtm. lets file a follow on ticket?

Reply:
Should fall into the scope of https://github.com/eto-ai/rikai/issues/356

Reply:
fine by me

Reply:
but let's add more details to #356 so we know what the issue actually is talking about